libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/NavigationRouteCallback.kt

```diff
@@ -6,9 +6,9 @@ import retrofit2.Callback
 import retrofit2.Response
 
 internal class NavigationRouteCallback @JvmOverloads constructor(
-    private val telemetry: NavigationTelemetry = NavigationTelemetry.getInstance(),
     private val listener: NavigationRouteEventListener,
-    private val callback: Callback<DirectionsResponse>
+    private val callback: Callback<DirectionsResponse>,
```

Comment:
Do we need to change the parameters order? `NavigationRouteCallbackTest`s changed because of this 🤔